utils/dhutil

```diff
@@ -294,9 +299,53 @@ def mpmap(api, args):
                     mntmap.append([dmpath, mntpoint, args])
 
     if args.mkfs:
-        pool = mp.Pool()
-        list(pool.imap_unordered(mkfs, mntmap))
+#        pool = mp.Pool()
+#        list(pool.imap_unordered(mkfs, mntmap))
+        return mntmap
+    else:
+       return map
+
+def create_fio_template():
+    with open('fiotemplate','w') as f:
+       lines = ["[global]","randrepeat=0","ioengine=libaio","iodepth=2","direct=1","numjobs=2","size=1G","runtime=3600","group_reporting","time_based"]
+       try:
+          for line in lines:
+             f.write(line + '\n')
+       finally:
+          f.close()
+
+def create_fio_files(api,args):
+
+    fio = {'fio_randread':{'rw':'randread', 'blocksize':'4k'},
+           'fio_seqread':{'rw':'read', 'blocksize':'1m'},
+           'fio_randwrite':{'rw':'randwrite', 'blocksize':'4k'},
+           'fio_seqwrite':{'rw':'write', 'blocksize':'1m'}
+          }
 
+    create_fio_template()
+    mntmap = mpmap(api,args)
+
+    try:
+       for key,item in fio.items():
+          with open('fiotemplate', 'r') as f:
+             with open(key, 'w') as key:
+                for line in f:
+                   key.write(line)
+                for param in item:
+                   key.write(param + "=" + item[param] + '\n')
+                if args.mkfs:
+                   for index in range(len(mntmap)):
+                      key.write("[" + str(key).split()[2].strip(',') + "]" + '\n')
+                      key.write("directory=/" + mntmap[index][1] + '\n')
+                else:
+                    for id in mntmap:
+                       key.write("[" + str(key).split()[2].strip(',') + "]" + '\n')
+                       sd = iqn_to_sd(mntmap[id]['iqn'])
+                       if sd:
+                          dm = sd_to_dm(sd)
+                          key.write("filename=/dev/" + dm + '\n')
```

Comment:
In order to make it easier to support a mix of volume sizes, instead of adding the device names to the global section, we might want to do something like this:

[global]
...
time_based

[job1]
filename=/dev/dm-1
size=100%

[job2]
filename=/dev/dm-2
size=100%